config/activator-ds.yaml

```diff
@@ -0,0 +1,104 @@
+# Copyright 2020 The Knative Authors
```

Comment:
Most all of the config was recently moved out of config to provide some structure. Should this not go under one of the subdirectories?

Reply:
this is done :)